packages/orchestration/src/typeGuards.js

```diff
@@ -222,13 +222,14 @@ harden(TimestampProtoShape);
  *
  * @internal
  */
-export const TxBodyOptsShape = M.splitRecord(
+export const ExecuteICATxOptsShape = M.splitRecord(
   {},
   {
     memo: M.string(),
     timeoutHeight: M.bigint(),
     extensionOptions: M.arrayOf(M.any()),
     nonCriticalExtensionOptions: M.arrayOf(M.any()),
+    relativeTimeoutNs: M.bigint(),
```

Comment:
```suggestion
    sendOpts: SendOptionsShape,
```

Where somewhere in `@agoric/network` exports:

```js
export const SendOptionsShape = M.splitRecord(
  {},
  {
    relativeTimeoutNs: M.bigint(),
  },
);
```